controllers/applicationvolumereplication_controller.go

```diff
@@ -116,19 +115,33 @@ func (r *ApplicationVolumeReplicationReconciler) Reconcile(ctx context.Context,
 	if err := r.updateAVRStatus(ctx, avr, placementDecisions); err != nil {
 		logger.Error(err, "failed to update status")
 
-		return ctrl.Result{}, err
+		return ctrl.Result{Requeue: true}, err
 	}
 
 	logger.Info("completed manifestwork for subscriptions")
 
-	return ctrl.Result{}, nil
+	return ctrl.Result{Requeue: requeue}, nil
 }
 
 func (r *ApplicationVolumeReplicationReconciler) processSubscriptionList(
-	subscriptionList *subv1.SubscriptionList) ramendrv1alpha1.SubscriptionPlacementDecisionMap {
+	subscriptionList *subv1.SubscriptionList) (
+	ramendrv1alpha1.SubscriptionPlacementDecisionMap, bool) {
 	placementDecisions := ramendrv1alpha1.SubscriptionPlacementDecisionMap{}
 
+	requeue := false
+
 	for _, subscription := range subscriptionList.Items {
+		// On the hub ignore any managed cluster subscriptions, as the hub maybe a managed cluster itself.
+		// SubscriptionSubscribed means this subscription is child sitting in managed cluster
+		// Placement.Local is true for a local subscription, and can be used in the absence of Status
+		if subscription.Status.Phase == subv1.SubscriptionSubscribed ||
```

Comment:
Updated this condition for subscriptions that do not have status updated (happened a couple of times during my testing)